index.js

```diff
@@ -35,4 +37,37 @@ fetch(FORUM_API)
     isLoading = false;
   });
 
+const displayTopicList = (topicList) => {
+  for (const topic of topicList) {
+    const categoryId = topic["category_id"];
+    if (!(categoryId in forumCategoriesObj)) {
+      continue;
+    }
+    const category = forumCategoriesObj[categoryId].className;
+    const categoryText = forumCategoriesObj[categoryId].category;
+    displayTopic(topic, category, categoryText);
+  }
+};
```

Comment:
I tried to make the code more explicit and readable.
Also, since you're passing the `topic` object into the `displayTopic` function
I think it is not necessary to pass the other arguments as it can be dealt with inside the `displayTopic`
Use built-in method whenever you can.

```suggestion
const displayTopicList = (topicList) => {
    topicList
    .filter(topic => topic["category_id"] in supportedTopicCategories)
    .forEach(displayTopic);
};
```

Reply:
Oh, of course.